OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -96,28 +99,28 @@ void TracerThread::Run(
     perf_event_enable(fd_to_ring_buffer.first);
   }
 
-  // Record and periodically print basic statistics on the number events.
-  constexpr uint64_t EVENT_COUNT_WINDOW_S = 5;
-  uint64_t event_count_window_begin_ns = 0;
-  uint64_t sched_switch_count = 0;
-  uint64_t sample_count = 0;
-  uint64_t uprobes_count = 0;
+  event_count_window_begin_ns_ = 0;
+  sched_switch_count_ = 0;
+  sample_count_ = 0;
+  uprobes_count_ = 0;
 
   bool last_iteration_saw_events = false;
 
+  std::thread defered_events_thread(&TracerThread::ProcessDeferredEvents, this,
+                                    exit_requested);
+
   while (!(*exit_requested)) {
     // Wait if there was no new event in the last iteration, so that we are not
     // constantly polling the buffers. 10 ms are still small enough to not have
```

Comment:
Change the comment to reflect that we use 1 ms now?

Reply:
I'd suggest to remove any explicit mention of the value in the comment, o/w this will happen again (code / comment getting out of sync). I'd just say something like: "We pick a value small enough to not have..."